.github/workflows/test-e2e-cli.yml

```diff
@@ -2,6 +2,10 @@ name: 'E2E Tests: CLI'
 
 on:
   workflow_call:
+    inputs:
+      version:
+        type: string
+        required: false
```

Comment:
By default je `version` v E2E testoch `dev`, pri CLI release ju nastavime na version z release, aby sme to lepsie otestovali.